### ISSUE TITLE
ARIA attributes, drawer should act as modal when overlay mode

### ISSUE THREAD

**Issue by alvarezguille** [vaadin/vaadin-app-layout#65]
Screen readers should announce the content on the drawer when it's open on overlay mode

**Comment by DiegoCardoso** [comment]
Some reference on the subject:
https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html
